.gitignore

```diff
@@ -18,8 +18,7 @@ pubspec.lock
 *.md~
 
 # IDEs
-.idea
-*.iml
+.idea/
```

Comment:
There were a few files in there that we didn't want (some of the metadata is common for the project, and some is user and machine specific).